install/ersatztv-install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck
+# Co-Author: MickLesk (Canbiz)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/ErsatzTV/ErsatzTV
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies (Patience)"
+$STD apt-get install -y --no-install-recommends \
+  ffmpeg \
+  build-essential \
+  unzip \
+  pkg-config \
+  curl \
+  sudo \
+  git \
+  make \
+  mc
+msg_ok "Installed Dependencies"
+
+
+msg_info "Installing ErsatzTV (Patience)" 
+cd /opt
+RELEASE=$(curl -s https://api.github.com/repos/ErsatzTV/ErsatzTV/releases | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }' | head -n 1)
+wget -q https://github.com/ErsatzTV/ErsatzTV/releases/download/${RELEASE}/ErsatzTV-${RELEASE}-linux-x64.tar.gz
+tar -xf ErsatzTV-${RELEASE}-linux-x64.tar.gz 
+mv ErsatzTV-${RELEASE}-linux-x64 ErsatzTV
+rm -R ErsatzTV-${RELEASE}-linux-x64.tar.gz 
+msg_ok "Installed ErsatzTV"
```

Comment:
I think that
```
msg_info "Installing ErsatzTV (Patience)" 
RELEASE=$(curl -s https://api.github.com/repos/ErsatzTV/ErsatzTV/releases | grep -oP '"tag_name": "\K[^"]+' | head -n 1)
wget -qO- "https://github.com/ErsatzTV/ErsatzTV/releases/download/${RELEASE}/ErsatzTV-${RELEASE}-linux-x64.tar.gz" | tar -xz -C /opt
mv "/opt/ErsatzTV-${RELEASE}-linux-x64" /opt/ErsatzTV
msg_ok "Installed ErsatzTV"
```
would be even better. What are your thoughts?

Reply:
very clean, i test it locally with my proxmox & when it work's i commit it immediately